### ISSUE TITLE
Support ISO9660 files

### ISSUE THREAD

**Issue by LubosD** [darlinghq/darling-dmg#34]
Consider supporting ISO9660 files, since many .dmg files are in fact .iso files.

This is also quite useful for Darling's hdiutil in general.

**Comment by tomkohen** [comment]
**LubosD** there're also FAT32 dmg images (created from hard drives with disk utility)

**Comment by LubosD** [comment]
Oh my... do you have any of those handy?

**Comment by kleuter** [comment]
No, but it can be easily created with Disk Utility on Mac, for example from a USB flash drive, also with compression

**Comment by tomkohen** [comment]
No, but it can be easily created with Disk Utility on Mac, for example from a USB flash drive, also with compression

**Comment by LubosD** [comment]
Do you think any of these are in the wild? Meaning someone is actually distributing such dmgs?

**Comment by tomkohen** [comment]
Well, not regularly. You may not find in Internet, but users sometimes backup their drives using Disk Utility (as they don't have much alternatives on OS X)

**Comment by tomkohen** [comment]
At least darling-dmg could offer some handling/fallback for such files

**Comment by tomkohen** [comment]
A good piece of code to implement ISO 9660/UDF right aray:
https://github.com/kindahl/ckfilesystem
